pkg/virt-controller/watch/migration.go

```diff
@@ -435,6 +438,33 @@ func (c *MigrationController) createTargetPod(migration *virtv1.VirtualMachineIn
 	return nil
 }
 
+func (c *MigrationController) createPDB(key string, vmi *virtv1.VirtualMachineInstance, vmim *virtv1.VirtualMachineInstanceMigration) error {
+	minAvailable := intstr.FromInt(2)
+	_, err := c.clientset.PolicyV1beta1().PodDisruptionBudgets(vmi.GetNamespace()).Create(context.Background(), &v1beta1.PodDisruptionBudget{
+		ObjectMeta: v1.ObjectMeta{
+			OwnerReferences: []v1.OwnerReference{
+				*v1.NewControllerRef(vmi, virtv1.VirtualMachineInstanceGroupVersionKind),
```

Comment:
can we get both an event on the VMI indicating the PDB has been created to protect the migration, and a log message indicating why the pdb is created similar to what i suggested in the pdb controller

Reply:
Done as well :)